### ISSUE TITLE
PHP 7.2 breaks unmanaged dependency

### ISSUE THREAD

**Issue by stephenswat** [shibdib/Keepstar#16]
While installing Keepstar on a machine with PHP 7.2 I noticed that the software does not work. After a little bit of investigation I realised that this is because Guzzle 6.2 is included in `/libraries/` (without being managed by Composer). PHP 7.2 changes the `count` function which means Guzzle 6.2 is incompatible with it (https://github.com/guzzle/guzzle/pull/1686). Guzzle >= 6.3.0 is compatible.

Please, if you can, either add a note that Keepstar requires PHP 7.0 or 7.1 (and not >=7.2), remove the unmanaged dependency and have it managed by Composer or update the version of Guzzle included in the source tree.

**Comment by ppfeufer** [comment]
With PHP7.0 being end of life and 7.1 out of maintenance, I suggest updateing Guzzle is the ideal and only way to solve this issue

https://secure.php.net/supported-versions.php

**Comment by steven-cmy** [comment]
![20190306212534](https://user-images.githubusercontent.com/4248629/53934177-8983eb80-4056-11e9-8b0e-7c50d6a4ecde.png)
can confirm this.

**Comment by Borst92** [comment]
Hey guys,
can anyone help me solving this issue? I tried to update Guzzle, but it won't work, so i got no clue how to do it. Is there a way to do it with composer?
Thanks alot!

**Comment by Neodork** [comment]
I agree with @StephenSwat for adding the guzzle dependency in composer.

I did it in my fork (currently no time for PR). And it works fine, you just need to remove the additional autoloader line.

**Comment by w33ts** [comment]
@Neodork Where is this additional autoloader line location?

**Comment by Neodork** [comment]
@DashieSystems Think I eventually decided to not do that but instead just add the library using composer instead. You can check the fork I have in my profile. It's been a long time since I implemented this solution.
